Sources/PackageModel/ManifestSourceGeneration.swift

```diff
@@ -319,8 +319,11 @@ fileprivate extension SourceCodeFragment {
             }
             self.init(enum: "target", subnodes: params)
             
-        case .product(name: let name, package: let packageName, condition: let condition):
+        case .product(name: let name, moduleAliases: let aliases, package: let packageName, condition: let condition):
```

Comment:
should we use the opportunity to clean up the syntax here?

eg `.product(let name, let aliases, let packageName, let condition)`

Reply:
sg